R/value-box.R

```diff
@@ -85,90 +168,128 @@ value_box_dependency_sass <- function(theme) {
   component_dependency_sass(theme, "value_box")
 }
 
-#' @param width one of the following:
+#' @param width,width_full_screen one of the following:
 #'   * A proportion (i.e., a number between 0 and 1) of available width to
 #'     allocate to the showcase.
-#'   * A vector of length 2 valid [CSS unit][htmltools::validateCssUnit] defining
-#'     the width of each column (for `showcase_left_center()` the 1st unit defines
-#'     the showcase width and for `showcase_top_right` the 2nd unit defines the
-#'     showcase width). Note that any units supported by the CSS grid
-#'     `grid-template-columns` property may be used (e.g., `fr` units).
+#'   * A valid [CSS unit][htmltools::validateCssUnit] defining the width of the
+#'     showcase column, or a valid value accepted by the `grid-template-columns`
+#'     CSS property to define the width of the showcase column. Accepted values
+#'     in the second category are `"auto"`, `"min-content"`, `"max-content"`, a
+#'     fractional unit (e.g. `2fr`), or a `minmax()` function (e.g.,
+#'     `minmax(100px, 1fr)`).
```

Comment:
Note to self: let's make sure whatever we're doing here also makes sense for py-shiny.